README.md

```diff
@@ -154,7 +154,7 @@ subproject.
 
 ### Microsoft Visual Studio
 
-The build files for Microsoft Visual Studio are generated for Visual Studio 2010.
+The build files for Microsoft Visual Studio are generated for Visual Studio 2012.
```

Comment:
Can we write it as `The build files are generated for Visual Studio 2012 or later`?

Reply:
Is it 2012 or 2013?

Reply:
I don't think so. 
Actually, looking at the [solution file](https://github.com/ARMmbed/mbedtls/blob/development/visualc/VS2010/mbedTLS.sln) it is still for 2010
(It will probably not work without having VS automatically update to the relevant version).
We should probably update our VS files

Reply:
@Patater yes. It is 2013. The VS version is 12, hence the confusion. I'll update the first change, and revert this change until the solution file is updated

Reply:
created #2962 to track this part